src/Generator/Generators/CSharp/CSharpCommentPrinter.cs

```diff
@@ -138,6 +138,13 @@ private static string FormatComment(List<Section> sections, CommentKind kind)
         {
             var commentPrefix = Comment.GetMultiLineCommentPrologue(kind);
             var commentBuilder = new StringBuilder();
+
+            sections = sections.OrderBy(s => s.Type).ToList();
+            var remarks = sections.Where(s => s.Type == CommentElement.Remarks).ToList();
+            if (remarks.Any())
+                remarks.First().GetLines().AddRange(remarks.Skip(1).SelectMany(s => s.GetLines()));
+            sections = sections.Except(remarks.Skip(1)).ToList();
```

Comment:
Again, do not assign. Instead, just remove the unnecessary ones. You could use RemoveRange.